src/psyclone/transformations.py

```diff
@@ -1032,15 +1035,23 @@ def apply(self, node, options=None):
         :type options: Optional[Dict[str, Any]]
 
         '''
+        if not options:
+            options = {}
+
         self.validate(node, options=options)
 
-        colours_loop = self._create_colours_loop(node)
+        tiled_colouring = options.get("tiling", False)
```

Comment:
In the "you touch it, you fix it" vein, please could you change this to use keyword arguments.